pilot/pkg/model/validation.go

```diff
@@ -39,9 +39,9 @@ import (
 
 const (
 	dns1123LabelMaxLength int    = 63
-	dns1123LabelFmt       string = "[a-zA-Z0-9]([-a-z-A-Z0-9]*[a-zA-Z0-9])?"
+	dns1123LabelFmt       string = "[a-z0-9]([-a-z0-9]*[a-z0-9])?"
 	// a wild-card prefix is an '*', a normal DNS1123 label with a leading '*' or '*-', or a normal DNS1123 label
-	wildcardPrefix string = `\*|(\*|\*-)?(` + dns1123LabelFmt + `)`
+	wildcardPrefix string = `(\*|(\*|\*-))?(` + dns1123LabelFmt + `)`
```

Comment:
why change this? its currently * or *dns1123label..
what you have is
(* or * or *-)dns1123label

Reply:
The previous regex can match `a/b/c` and many other invalid format